api/types.d.ts

```diff
@@ -5159,6 +5162,14 @@ export interface CatIndicesRequest extends CatCatRequestBase {
 
 export type CatIndicesResponse = CatIndicesIndicesRecord[]
 
+export interface CatClusterManagerClusterManagerRecord {
+  id?: string
+  host?: string
+  h?: string
+  ip?: string
+  node?: string
+  n?: string
+}
 export interface CatMasterMasterRecord {
```

Comment:
You mean like this:
```
export interface CatBaseRecord {
  id?: string
  host?: string
  h?: string
  ip?: string
  node?: string
  n?: string
}

export interface CatClusterManagerClusterManagerRecord extends CatBaseRecord {}

export interface CatMasterMasterRecord extends CatBaseRecord {}
```

Reply:
Yeah, that would work fine.

Reply:
I ended up just adding comments since this will make less changes in the future. We will just do bunch of deletion and cleans. no need to worry about any functional changes.